garnett/trajectory.py

```diff
@@ -1280,12 +1361,12 @@ def _generate_type_id_array(types, type_ids):
 
 def copyto_hoomd_blue_snapshot(frame, snapshot):
     "Copy the frame into a HOOMD-blue snapshot."
-    if frame.positions is not None:
-        np.copyto(snapshot.particles.position, frame.positions)
-    if frame.orientations is not None:
-        np.copyto(snapshot.particles.orientation, frame.orientations)
-    if frame.velocities is not None:
-        np.copyto(snapshot.particles.velocity, frame.velocities)
+    if frame.position is not None:
```

Comment:
Similar to the GSD reader/writer above, I would simplify this:

```python
for prop in ['position', ...]:
    data = getattr(frame, prop, None)
    if data is not None:
        np.copyto(getattr(snapshot.particles, prop), data)
```

Reply:
Missed simplification